components/UI/Button.tsx

```diff
@@ -1,6 +1,11 @@
 import React from "react";
 
-export default function Button({ children, styles }) {
+type ButtonProps = {
```

Comment:
use interface instead of type, because other team members are using interface

Reply:
ok